src/aioquic/quic/connection.py

```diff
@@ -2718,3 +2758,33 @@ def _write_streams_blocked_frame(
                     limit=limit,
                 )
             )
+
+    def _write_stream_reset_frame(
+        self,
+        builder: QuicPacketBuilder,
+        frame_type: QuicFrameType,
+        stream: QuicStream,
+        error_code: int
+        ) -> None:
+
+        buf = builder.start_frame(
+            frame_type=frame_type,
+            capacity= RESET_STREAM_CAPACITY,
+            handler=self._on_reset_stream_delivery,
+        )
+        buf.push_uint_var(stream.stream_id)
+        buf.push_uint_var(error_code)
+
+        length = len(size_uint_var(stream.stream_id))
+        final_size = size_uint_var(stream.stream_id) + length
```

Comment:
I don't think this is correct, I believe final_size represents how much data has been sent on the stream

Reply:
Is `_send_highest` attribute from QuicStream which denotes the final size?